implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java

```diff
@@ -369,12 +369,12 @@ private void fillBaseName(StringBuilder sb, MetricRegistry.Type scope, String ke
             sb.append(suffix);
     }
 
-    private void writeHelpLine(StringBuilder sb, MetricRegistry.Type scope, String key, Metadata md, String suffix) {
+    private void writeHelpLine(final StringBuilder sb, MetricRegistry.Type scope, String key, Metadata md, String suffix) {
         // Only write this line if we actually have a description in metadata
         if (writeHelpLine && !md.getDescription().orElse("").isEmpty() && !alreadyExportedNames.get().contains(md.getName())) {
             sb.append("# HELP ");
             getNameWithScopeAndSuffix(sb, scope, key, suffix);
-            sb.append(quoteHelpText(md.getDescription().get()));
+            md.getDescription().ifPresent(s -> sb.append(quoteHelpText(s)));
```

Comment:
There is always a non-empty description here thanks to the condition above
> !md.getDescription().orElse("").isEmpty()

so having this `ifPresent` here is a bit confusing (it is always present)
It would be a bit nicer if we extracted the value of the description before the `if` condition, something like
```
String description = md.getDescription().orElse("");
```
and changed the 
```
!md.getDescription().orElse("").isEmpty()
```

condition to 
```
!description.isEmpty()
```
to make sure that there's a non-empty string. WDYT?

Reply:
fixed